src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+/* eslint-disable no-nested-ternary */
+import './Person.scss';
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <div className="Person">
+      <p className="Person__name">My name is {name}</p>
+      {age && <p className="Person__age">I am {age}</p>}
+      {isMarried ? (
+        sex === 'm' ? (
+          <p className="Person__partner">{partnerName} is my wife</p>
+        ) : (
+          <p className="Person__partner">{partnerName} is my husband</p>
+        )
+      ) : (
+        <p className="Person__partner">I am not married</p>
+      )}
```

Comment:
try to avoid nested conditionals
create fn or variable for instance - for getting sex